src/main/java/org/checkerframework/specimin/TargetMethodFinderVisitor.java

```diff
@@ -330,4 +328,16 @@ public Visitable visit(NameExpr expr, Void p) {
     }
     return super.visit(expr, p);
   }
+
+  /**
+   * @param type unionType parameter
```

Comment:
Please provide a Javadoc that summarizes the purpose of the method for every method that you write (the build system should also warn you that this is missing)

Reply:
build system did not show any warning for this method. I guess method description is not forced. I added method description. Added documentation for a method that was not part of the PR

Reply:
> build system did not show any warning for this method. I guess method description is not forced

That's probably a misconfiguration in the build system. I'll look into it later today.

> Added documentation for a method that was not part of the PR

That shouldn't be necessary (now or ever). But, you should do so for new code.